### PR TITLE
'Add to email list' SendGrid function added (both ES6 and old fashioned js).

### PR DESCRIPTION
**Pull request by heymartinadams**
With awesome help from @kbrandwijk!

### CODE REVIEW

**Comment by kbrandwijk** [comment]
@heymartinadams Can you update `let` to `const` in the ES6 example for constants (like `sg`)? Also, have you tested the sg promise wrapper resolve/reject syntax?

File: sendgrid-email/addtolist_es6.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ module.exports = (event) => {
 
   return new Promise((resolve, reject) => {
     var request = sg.emptyRequest()
-    const { email, first_name: nameFirst, last_name: nameLast } = event.data.User.node
-    request.body = [{email, first_name, last_name }];
+    const { email, nameFirst, nameLast } = event.data.User.node
+    request.body = [{email, first_name: nameFirst, last_name: nameLast }];
```
Comment (kbrandwijk) [inline sendgrid-email/addtolist_es6.js]: Can you try `request.body = [({ email, first_name, last_name } = event.data.User.node)]`?

Comment (heymartinadams) [inline sendgrid-email/addtolist_es6.js]: Hi @kbrandwijk. The syntax `{ email, first_name, last_name } = event.data.User.node` doesn’t make sense if the schema is `nameFirst`, `nameLast`, etc, no?

Comment (kbrandwijk) [inline sendgrid-email/addtolist_es6.js]: You're right. Changed it. it's about combining line 7 and 8.

Comment (heymartinadams) [inline sendgrid-email/addtolist_es6.js]: I tried this:
```
const { email, first_name: nameFirst, last_name: nameLast } = event.data.User.node
request.body = [{email, first_name, last_name }];
```
but the compiler wouldn’t allow it.

Comment (kbrandwijk) [inline sendgrid-email/addtolist_es6.js]: It's the other way around:
```
const { email, nameFirst: first_name, nameLast: last_name } = event.data.User.node
request.body = [{ email, first_name, last_name }]
```

Comment (heymartinadams) [inline sendgrid-email/addtolist_es6.js]: Woah... learned something new again. Thanks, @kbrandwijk

**Comment by marktani** [comment]
you guys rock 🎸 💯

**Pull request merged**